test/openAPI/features/client_create.feature

```diff
@@ -1,16 +1,130 @@
-Feature: API to add new open ID connect (OIDC) clients, it can be invoked by other modules which manage the relying parties/partners.
+@method=POST @endpoint=/client-mgmt/oidc-client
```

Comment:
Enpoint is not match the PR title

Reply:
In this case I also needed create a new branch from existing https://github.com/GovStackWorkingGroup/bb-identity/pull/27 this is why this file is also here. Please add review only for oidc_authorize.feature ;)